src/config.ts

```diff
@@ -201,11 +203,11 @@ export const chainsData = {
 } as const
 
 export const liveHostAddress = '0x67Fca88E2f5F2C33b86bFa4EccfCb8dCD6a56D17'
-export const liveSupportedChains: ChainId[] = [ gnosis.id, polygonAmoy.id ]
+export const liveSupportedChains: ChainId[] = [ polygon.id, gnosis.id, polygonAmoy.id ]
 /**
- * Live Betting for Gnosis in beta, only 1 WXDAI possible
+ * Live Betting in beta
  */
-export const liveBetAmount = '1'
+export const minLiveBetAmount = 1
```

Comment:
Because it may be used in apps, it's better not to do "breaking change", just deprecate liveBetAmount:
```js
/**
 * @deprecated Live bets are not strictly limited to "1" anymore.\
 * Instead, use **`minLiveBetAmount`** for min bet value,\
 * or **`minBet`** + **`maxBet`** parameters from **`useDetailedBetslip()`**.
 */
export const liveBetAmount = 1

export const minLiveBetAmount = 1
```

Reply:
e4b25810e6cc1c2afacd57679605dc48bbb353d3